doc/source/whatsnew/v0.13.0.rst

```diff
@@ -184,7 +184,7 @@ API changes
   .. ipython:: python
      :okwarning:
 
-     dfc.loc[0]['A'] = 1111
+     dfc.loc[0]['B'] = 1111
```

Comment:
The B column are already integers, so this example of chained assignment is still valid this way, but then doesn't include dtype changes